src/clj_irods/jargon.clj

```diff
@@ -0,0 +1,83 @@
+(ns clj-irods.jargon
```

Comment:
I think this namespace is pretty understandable in the context of the rest, hopefully.